File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171704/Business/Persona.cs
using System.Collections.Generic;

namespace Business
{
    public class Persona
    {
	    private static Data.Persona obj = new Data.Persona();

        public static int Add(Models.Persona value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.Persona value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.Persona GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.Persona GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.Persona> GetAll()
        {
            return obj.GetAll();
        }
    }
}

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/Persona.vb
Imports System.Collections.Generic

Public Class Persona
    Public Shared Function Add(value As Models.Persona) As Integer
        Return Data.Persona.Add(value)
    End Function

    Public Shared Function Edit(value As Models.Persona) As Integer
        Return Data.Persona.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.Persona.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.Persona
        Return Data.Persona.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.Persona
        Return Data.Persona.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.Persona)
        Return Data.Persona.GetAll()
    End Function
End Class